adal/build.gradle

```diff
@@ -138,44 +150,6 @@ task jacocoTestReport(type: JacocoReport, dependsOn: 'testDebugUnitTest') {
     ])
 }
 
-task createPom  {
```

Comment:
We will need the Pom file for the official release in both public maven and vsts maven. Not sure why we remove the pom task here.

Reply:
We will fix this in #1185